docs/source/modules/simcloud.rst

```diff
@@ -0,0 +1,65 @@
+Cloud Scheme (SimCloud)
+=======================
+
```

Comment:
Given the code uses other peoples code (i.e. CAM and Romps, was there anyone else?) I think we should credit that code in here somewhere.